go.mod

```diff
@@ -22,7 +22,6 @@ require (
 	github.com/gorilla/securecookie v1.1.1
 	github.com/gorilla/sessions v1.2.1
 	github.com/gorilla/websocket v1.4.2
-	github.com/ipfs/go-datastore v0.4.5 // indirect
```

Comment:
this is not actually part of this pr, yeah?

Reply:
nah just a mod tidy